.github/workflows/generate.yaml

```diff
@@ -6,7 +6,7 @@ on:
     - cron: '5 4 * * SUN'
 
 env:
-  SWAGGER_URL: ./.github/workflows/assets/swagger.json
+  SWAGGER_URL: ./.github/workflows/assets/swagger.yaml
```

Comment:
The documentations example uses a `.yaml` so we should be safe here:

https://github.com/OpenAPITools/openapi-generator-cli#usage